### PR TITLE
Verbose mode for ssh| scp commands

### PR DESCRIPTION
**Pull request by mimi1vx**

### CODE REVIEW

File: src/repose-add.zsh [hunk 1]
```diff
@@ -22,10 +22,12 @@ declare -gr cmdhelp=$'
...
-    -n,--print            Display, do not perform destructive commands
-    -t,--tag              Set default tags (standard are gm up lt se)
+    -h                     Display this message
+    --help                 Display full help
+    -n, --print            Display, do not perform destructive commands
+    -t, --tag              Set default tags (standard are gm up lt se)
+    -v, --verbose          Enable verbose mode for scp,ssh commands
+
   Operands:
     HOST                  Machine to operate on
```
Comment (rwmanos) [inline src/repose-add.zsh]: Get second column in line with above like you do elsewhere.

**Comment by rwmanos** [comment]
The "verbose" option was not added in repose-remove. Was it on purpose? It still uses ssh, doesn't it?

**Comment by mimi1vx** [comment]
@rwmanos  is added , only I forgot to update docu ///

**Comment by rwmanos** [comment]
OK, I added it to the man pages. Will you change the src file or should I?

Also, I changed the text from:
"Enable verbose mode for scp,ssh commands" to "Disable quiet mode for ssh and scp that suppresses most warning and diagnostic messages." because this is what it actually does.

I'll update all the src files in #74 , OK?

**Comment by mimi1vx** [comment]
@rwmanos  simple ? NO 

please dont mix  small src changes with this B I G docu PR

**Pull request merged**